pom.xml

```diff
@@ -83,33 +80,10 @@
       <version>31.1-jre</version>
     </dependency>
     <dependency>
-      <groupId>com.github.zafarkhaja</groupId>
-      <artifactId>java-semver</artifactId>
-      <version>0.9.0</version>
-    </dependency>
-
-    <dependency>
-      <groupId>org.graalvm.sdk</groupId>
-      <artifactId>graal-sdk</artifactId>
-      <version>${graalvm.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.graalvm.js</groupId>
-      <artifactId>js</artifactId>
-      <version>${graalvm.version}</version>
-      <scope>runtime</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.graalvm.js</groupId>
-      <artifactId>js-scriptengine</artifactId>
-      <version>${graalvm.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.skyscreamer</groupId>
-      <artifactId>jsonassert</artifactId>
-      <version>1.5.1</version>
-      <scope>test</scope>
+      <groupId>com.vdurmont</groupId>
+      <artifactId>semver4j</artifactId>
```

Comment:
I believe the repo maintained by vdurmont is not maintained, but the github org `semver4j` is carrying it on. According to their README.md, this is how we reference in maven:

```
<dependency>
    <groupId>org.semver4j</groupId>
    <artifactId>semver4j</artifactId>
    <version>4.3.0</version>
</dependency>
```
Looks like it is up to version 4.3.0 under the semver4j org.

Reply:
I've switched them round, this has its own perculiarites, but a bug in the other lib is fixed here so that is good :-)